src/tim/CMakeLists.txt

```diff
@@ -16,7 +24,13 @@ list(APPEND SRC
 include_directories(${PROJECT_SOURCE_DIR}/include)
 include_directories(${PROJECT_SOURCE_DIR}/include/tim/vx)
 include_directories(${PROJECT_SOURCE_DIR}/src/tim/vx)
-include_directories(${PROJECT_SOURCE_DIR}/src/tim/vx/internal/include)
+if(${TIM_VX_USE_EXTERNAL_OVXLIB})
+    set(OVXLIB_INCLUDE_DIR "${CMAKE_SYSROOT}/usr/include/OVXLIB")
```

Comment:
Can we setup ${OVXLIB_INC} instead of using a hard-coded path?

Reply:
Agree, done.